lib/mugshot.js

```diff
@@ -8,7 +8,7 @@ var LooksSameAdapter = require('./adapters/looks-same.js');
  * @type {Differ}
  * @const
  */
-var DEFAULT_DIFFER = new LooksSameAdapter();
+var DEFAULT_DIFFER = new LooksSameAdapter({tolerance: 1});
```

Comment:
It is 1%, but we decided to drop this and let the user choose what tolerance is needed.